apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -5,7 +5,37 @@ import React from 'react';
 
 import { connect } from 'react-redux';
 
-const TTS_URL = "https://tts.code.org/sharon22k/180/100";
+// TODO (elijah): have these constants shared w/dashboard
```

Comment:
it's been noted; I plan to come up with a plan for that once we decide how many more languages we're going to support with TTS